bin/node/bench/src/import.rs

```diff
@@ -155,6 +158,41 @@ impl core::Benchmark for ImportBenchmark {
 		context.import_block(self.block.clone());
 		let elapsed = start.elapsed();
 
+		// Sanity checks.
+		context.client.state_at(&BlockId::number(1)).expect("state_at failed for block#1")
+			.inspect_with(|| {
+				match self.block_type {
+					BlockType::RandomTransfersKeepAlive => {
+						// should be 5 per signed extrinsic + 1 per unsigned
```

Comment:
are those sanity checks correct @shawntabrizi ?

btw, do we want to write less events on transfer?
we can merge (1, 2) events into one for sure (like `RawEvent::EndowNew`)